index.html

```diff
@@ -5,9 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
 <body>
-    <header></header>
-    <section></section>
+    <header>
+        <div class="status-bar"></div>
+        <div class="toolbar">
+            <div class="logo-placeholder">
+                <p class="name white-color">Thomas Obuya</p>
+            </div>
+            <div class="menu-placeholder">
+                <button class="menu">
+                    <img src="menu.svg" alt="Menu">
+                </button>
+            </div>
+        </div>
+    </header>
+    <section class="headline">
+        <div class="headline-bio">
+            <p class="primary-text">Hey There. I'm Thomas</p>
+            <p class="supporting-text">I'am a Software Developer</p>
```

Comment:
- [ ] I will strongly recommend that you wrap the subtitle in a header tag `hx (such as <h2>,<h3>, etc...)` tag which will increase the accessibility of your website because search engines and screen readers will be able to understand what the website is all about quickly 😄 .